src/v3/bandwidthRtc.ts

```diff
@@ -0,0 +1,445 @@
+require("webrtc-adapter");
```

Comment:
The new hotness, there is commented out code that needs to be removed, some TODOs, etc. May need to add a new signal for reconnection as well. Comments welcome, but I wouldn't worry too much about some of the minor nitpicking, I'll be addressing that soon.